pkg/api/config/config.go

```diff
@@ -94,13 +94,19 @@ type OpenIDConfig struct {
 	Providers map[string]OpenIDProviderConfig
 }
 
-type OpenIDProviderConfig struct {
-	Name         string
+type OpenIDCredentials struct {
 	ClientID     string
 	ClientSecret string
-	KeyPath      string
-	Issuer       string
-	Scopes       []string
+}
+
+type OpenIDProviderConfig struct {
+	CredentialsFile string
+	Name            string
+	ClientID        string
```

Comment:
Might as well "inline" these as OpenIdCredentials?

Reply:
Do you think of

type OpenIDProviderConfig struct {
   ...
   OpenIDCredentials
   ...
}

I'm not sure, but I think that would force us to change all the fixtures in the tests and if we want to remove those fields some time in the future, the second commit can be just reverted ...